llvm/lib/MC/MCSectionELF.cpp

```diff
@@ -90,7 +90,9 @@ void MCSectionELF::printSwitchToSection(const MCAsmInfo &MAI, const Triple &T,
     OS << 'e';
   if (Flags & ELF::SHF_EXECINSTR)
     OS << 'x';
-  if (Flags & ELF::SHF_GROUP)
+  // TODO: Always print G after o to be clear that the 'G' argument is parsed
```

Comment:
Can you file an issue for this and reference it here so we don't loose track of this `TODO`?

Reply:
Thanks for the quick review. I have just created #77422 to clean up this and update some tests (not that many), but I want to keep them separate from this change.